<?xml version="1.0"?>
<launch>
	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" required="true">
		<rosparam file="$(find gobot_move)/params/costmap_common_params.yaml" command="load" ns="global_costmap"/>
		<rosparam file="$(find gobot_move)/params/costmap_common_params.yaml" command="load" ns="local_costmap"/>
    		<rosparam file="$(find gobot_move)/params/local_costmap_params.yaml" command="load"/>
    		<rosparam file="$(find gobot_move)/params/global_costmap_params.yaml" command="load"/>
		<rosparam file="$(find gobot_move)/params/move_base_params.yaml" command="load"/>
		<rosparam file="$(find gobot_move)/params/base_local_planner_params.yaml" command="load"/>
		<!--
		<rosparam file="$(find gobot_move)/params/global_planner_params.yaml" command="load"/>
		<rosparam file="$(find gobot_move)/params/navfn_global_planner_params.yaml" command="load"/>
		-->
  	</node>
</launch>
